### DISCUSSION TITLE
$$ Paid Assets Assessments $$

### DISCUSSION THREAD

**Discussion by jelster** [jelster/space-truckers#38] [Ideas]
Not every assets we want or need for the game is going to be available _gratis_. The project has a very small budget for licensing of models, textures, six, etc. so there's a big priority on ensuring it's spent in the right places. 

Post a link and images of a commercial asset along with the price and why you think it's worth consideration of purchase. 

Use thread replies to respond to a given post, write a new comment to add a new entry

**Comment by jelster** [comment]
# Space truck model

[This asset](https://sketchfab.com/3d-models/generic-americam-semi-truck-dbd6f36d34f1419dbf07ede48e174b66) is a candidate for what is probably going to be worth spending more on since its kind of the center piece for the game. 

![image](https://user-images.githubusercontent.com/1508500/107152350-0c8dda80-692d-11eb-92cf-f22b470702da.jpeg)

**pros**
- Game ready with low poly counts
- Low poly Interior cabin with cockpit
- PBR materials and textures provided 

**cons**
- Price would take up much of the assets budget
- Unknown effort needed for modifications

**Comment by jelster** [comment]
[# Vehicle low poly pack](https://sketchfab.com/3d-models/vehicle-pack-lowpoly-pbr-af7d4ea3034c4bd18156fd71ad0e86db)

This is a moderately priced asset pack that would potentially give us all the various road traffic we might need. 
![image](https://user-images.githubusercontent.com/1508500/107152547-254ac000-692e-11eb-8ba5-31a814e9fac9.jpeg)

- **good** a lot of variety
- **bad** is it really needed?

**Comment by jelster** [comment]
Semi truck rig candidate - [Peterbilt](https://sketchfab.com/3d-models/peterbilt-379-with-interior-aa9e661f7c9245e1ba016ce171e175fc)

![image](https://user-images.githubusercontent.com/1508500/107169622-f65f3900-6983-11eb-80c2-55d0f1dd7b2e.jpeg)

**Reply by jelster** [reply]
- Moderate polys
- Cab interior fully modeled
- A little less expensive than the generic American semi
- Designed for adaptability (according to the artist...)

**Comment by jelster** [comment]
[30 Flying cars pack](https://sketchfab.com/3d-models/12-fly-cars-pack-f676ac86e3384c6eb292809484bfbff0)

![image](https://user-images.githubusercontent.com/1508500/107170091-1511ff80-6985-11eb-93d2-080a2cd0333a.jpeg)

**Reply by jelster** [reply]
Ok have to admit I really like this one

- Game ready models of different types of cars w/ PBR
- Animated spinning lights 
- Modest price
- Ready to tweak
- Potential 👎 : may not fit thematically

**Comment by jelster** [comment]
# [Donut vending machine](https://sketchfab.com/3d-models/donut-vending-machine-f1b136499d614afc95e1349b8cf9dfc7)

Another 😍 🤤
![image](https://user-images.githubusercontent.com/1508500/107170680-9e760180-6986-11eb-9c51-8dc6897bae9f.jpeg)

**Reply by jelster** [reply]
- Picture is all the 👍 needed
- Modestly priced 
- potentially not performant without modification

**Comment by eldinor** [comment]
Didn't try yet but seems to be good for some signs - https://gfx-hub.cc/3d-models/collections/64064-blender-market-bs-signs-450-street-signs.html

**Reply by jelster** [reply]
Great to hear from you - that is a nice find! It's got nice weathering and such too -- I can see it being straight-forward to project arbitrary text onto a sign too!

I couldn't find any mention of how they're licensed and the site doesn't directly host the content so I'm not sure who the actual author is - would you happen to know that info for these assets, or at least licensing info?
